os/arch/arm/src/armv7-m/up_blocktask.c

```diff
@@ -158,6 +162,11 @@ void up_block_task(struct tcb_s *tcb, tstate_t task_state)
 			save_task_scheduling_status(rtcb);
 #endif
 
+			/* Restore the MPU registers in case we are switching to an application task */
+#ifdef CONFIG_ARMV7M_MPU
```

Comment:
Function definition is already in conditionals. Is this conditional necessary??
It looks unnecessary.

Reply:
We still need this condition because the function is defined in mpu.h under this condition.

Reply:
Then, you don't need to add ` defined(CONFIG_ARMV7M_MPU)` in function definition.

Reply:
ok. fixed